script/Ciel/Day 10/10_01_CIEL10_5.txt

```diff
@@ -772,82 +870,94 @@ But what she showed me after her fight with Arcueid was something even more 'imp
 -- あの地獄みたいな状況で、先輩はみんなを守る為に戦ってくれた。怖がるなんてとんでもない。俺はあの勇敢さに、今まで何度も励まされたんですから」
-You were fighting to protect all of us inside that hellish landscape. How could that be scary to me? On the contrary, your bravery has inspired me several times ever since."
+Despite those hellish conditions, you still fought to protect us.
+ Why on earth would that scare me away?
+ On the contrary, thinking back on your bravery has inspired me countless times."
 }
 [sha:b3b2b4100393d58e4ff8459b7758ee9d32aa25cb]{
 -- Page 343, Offset 27996.
 -- 本当は“格好いい”なんて浮ついた言葉じゃなく、尊敬していると伝えたかった。
-To be honest, the first thing that came to my mind wasn't that 'she looked cool' but that I wanted to convey to her that I respect her.
+If I'm being honest, I didn't really want to use a frivolous word like 'cool' to describe her――
+I wanted to more accurately convey just how much respect I have for her.
 }
 [sha:34f7efd3900b8bf46b50d895041946532b1cdd21]{
 -- Page 343, Offset 27997.
 -- でも、俺から“尊敬しています”なんて言われても先輩は困るだろうし、と自重する。
-But I didn't say it because I reckoned that she would have been embarrassed if I said something of the sort.
+Alas, I felt like she'd probably get embarrassed if I started talking about how much I respected her, so I held back.
 }
 [sha:9311de3da1d2ff514ec4aff09ad1d6cf56254f2c]{
 -- Page 344, Offset 27998.
 -- 「遠野くん、それは違いますよ。
-"Tohno-kun, you're wrong on that point.
+"I think you've misunderstood, Tohno-kun.
 }
 [sha:401261b8e6824c5b52c0c8e1bf6d639ecb9c6ad2]{
 -- Page 344, Offset 27999.
 -- 私は勇敢なんかじゃありません。死への恐怖がないだけなんです。だって私は死にませんから。あの時だって恐怖なんて欠片もなかった」
-I'm not brave or anything like that. I just don't fear death, that's all. After all, I can't die. I didn't feel one ounce of fear back then."
+I'm not brave or anything like that.
+ I simply don't fear death.
+ After all, I can't die.
+ Even during that fight, I didn't feel even an ounce of fear."
 }
 [sha:673ead276fe6ce5731181479e9fd1d0994b924f2]{
 -- Page 345, Offset 28000.
 -- 「……私はただ、鈍感なだけなんです。
-"I just... don't feel much.
+"I'm just insensitive to it, that's all.
 }
 [sha:76b5be1fbb73dfb59030e64bbca2ed71c4f73872]{
 -- Page 345, Offset 28001.
 -- 本当に勇気を持つ人間とは、死に怯えながらも戦える人の事です。
-The really courageous ones are the humans who fight despite their fear of death.
+Truly courageous people are the ones who continue to fight %{i}despite%{/i} being afraid of death.
 }
 [sha:2619094fb28dff110f7c35578bd2955ffcc3c399]{
 -- Page 345, Offset 28002.
 -- 私にその勇気はありません。だって死にませんから」
-I don't have such courage. Again, I can't die."
+I don't have that courage――I simply can't die."
 }
 [sha:e956a4f11bcf320e0d3928d664b56ee07e073086]{
 -- Page 346, Offset 28003.
 -- 「私が普通ではないのはこの精神性です。死への恐怖に関してだけ言えば、私は吸血鬼以上の化け物でしょう」
-"What's unusual with me is my mentality. If I had to judge solely in terms of how much one fears death, I'm probably even more of a monster than vampires."
+"It's this mindset that makes me so abnormal.
+ As far as a fear of death is anything to go by,
+ I'm probably even more of a monster than vampires are."
```

Comment:
"As far as a fear of death is anything to go by" sound highly odd.
Maybe "At least in terms of lacking a fear of death, I'm..."